scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
-**TODO** update this section
 # Configuring Supply Chain Security Tools - Policy
 
 This component requires extra configuration steps to start verifying your
 container images properly.
 
-The instructions in this section only apply to the deployment namespace of
-Supply Chain Security Tools - Sign. In most cases, this namespace is
-rendered as the default namespace `image-policy-system`.
+## <a id="admission-of-images"></a> Admission of Images
 
-If you deployed Supply Chain Security Tools - Sign by using a customized
-namespace specified in the installation values file, replace `image-policy-system`
-with the namespace name that you specified in `deployment_namespace` before
-performing the configuration steps.
+An image is admitted after it has been validated against all
+[ClusterImagePolicy](#create-cip-resource) that matched the digest of the image
+and that there was at least one valid signature or attestation obtained from the
+authorities provided in each of the matched ClusterImagePolicy. So each
+ClusterImagePolicy that matches is AND for admission, and within each
+ClusterImagePolicy authorities are ON.
```

Comment:
```
So all matched ClusterImagePolicies must be validated. For a ClusterImagePolicy to be valid, at least one authority in the ClusterImagePolicy must validate the image digest.
```

Or, if you want to keep the current sentences:
```
So each ClusterImagePolicy that matches will be an AND operation for admission, and within each ClusterImagePolicy, authorities are OR operated for admission.
```

Just thought it might read better by saying what the `AND` and `ON`, which I believe was supposed to be `OR`, means.

Reply:
I think all of these are a bit rough. What do you think of this?

> An image is admitted after it has been validated against all policies with
matching image patterns, and where at least one valid signature obtained from
the authorities provided in each of the matched
[ClusterImagePolicy](#create-cip-resource). Within a single policy, any single
signature must be valid. When more than one policy has a matching image pattern,
the image much match at least one signature from each ClusterImagePolicy.

Reply:
Hmm... to me I feel like there is a lot of repetition. How about:

>An image is admitted after it has been validated against all [ClusterImagePolicies](https://github.com/derwei/docs-tap/pull/1#create-cip-resource) with matching image patterns. >Within each matched policy, any single signature must be valid.